solutions/cf-839C.mdx

```diff
@@ -0,0 +1,106 @@
+---
+id: cf-839C
+source: CF
+title: Journey
+author: Reyansh Bahl
+---
+
+[Official Solution (C++)](https://codeforces.com/blog/entry/53815)
+
+## Explanation
+
+The first step is to calculate the length and probability of each possible path.
+We can do this using DFS. We start of with a probability of 1, or 100%. Then,
+for each node, we divide the current path's probability by the number of
+possible next moves.
+
+For example, if node 1 is connected to nodes 2 and 3, the probability of
+reaching node 2 is $1/2$, or $0.5$, and the probability of reaching node 3 is
+also $0.5$. If node 3 is connected to nodes 4 and 5, the probability of reaching
+node 4 is $0.5 / 2 = 0.25$ and the probability of reaching node 5 is also
```

Comment:
```suggestion
node 4 is $\frac{0.5}{2} = 0.25$ and the probability of reaching node 5 is also
```

Reply:
Since I didn't intend for 0.5 divided by 2 to be represented as a fraction, I think it would be better with the division symbol instead (I changed it to be the division symbol instead of forward slash).